src/lib/constants.ts

```diff
@@ -0,0 +1,8 @@
+export const ERROR_MESSAGE = {
+    NO_VAULTS_FOUND_BY_TITLE: "No Vaults found with title",
+    MULTIPLE_VAULTS_FOUND_BY_TITLE: "Found multiple Vaults with given title. Provide a more specific Item title",
```

Comment:
Should be ```Provide a more specific Vault title```. Though I'm also wondering if we should ask the user to provide a uuid here if they want to be more specific?

Reply:
done